docs/guides/server/hostname.adoc

```diff
@@ -40,7 +40,7 @@ Most of the time, it should be enough to set the `hostname` option in order to c
 When using the `hostname` option the server is going to resolve the HTTP scheme, port, and path, automatically so that:
 
 * `https` scheme is used unless you set `hostname-strict-https=false`
-* if the `proxy` option is set, the proxy will use the default ports (i.e.: 80 and 443).  If the proxy uses a different port, it needs to be specified via the `hostname-port` configuration option
+* if the `proxy-headers` option is set, the proxy will use the default ports (i.e.: 80 and 443).  If the proxy uses a different port, it needs to be specified via the `hostname-port` configuration option
```

Comment:
IMO, we should not recommend using `hostname-port` but using the `hostname-url` option.

This option does not much sense if you can just provide the URL you want to expose the frontend URLs (same thing for admin). The `hostname-port` is kinda another option causing the confusion you want to solve with the on-going changes in the hostname configuration and implementation.

Reply:
Done.